frontend/src/metabase/visualizations/components/settings/ChartSettingFieldsPicker.jsx

```diff
@@ -46,9 +47,9 @@ const ChartSettingFieldsPicker = ({
       <span className="text-error">{t`error`}</span>
     )}
     {addAnother && (
-      <div className="mt2 mb3">
+      <AddAnotherContainer>
         <a
-          className="text-brand text-bold py1 px2 rounded bg-light bg-medium-hover"
+          className="text-brand text-bold py1 rounded bg-light"
```

Comment:
That was intentional. Here is the screenshot from figma: 
![image](https://user-images.githubusercontent.com/1328979/189356519-dc7d241a-e5ef-4b76-b8e9-b193946c5689.png)

Reply:
Ok, this makes sense, but you can see that there's still barely a button background as opposed to just a link-styled button (without background color).

Here's how stats looks w/o hovering, there's still a background color around the button, so I think in your PR, the background should be removed, otherwise it will look broken.
![image](https://user-images.githubusercontent.com/1937582/189386371-ae8f83d6-d5db-464f-8d69-70bdd6b6c75b.png)

Reply:
True! I'll remove it